### PR TITLE
Precious/mech page

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

New mech page:
* Mech metrics (coming soon) section
* What is Olas Mech + process
* Why Olas Mech

## Screenshots/Recordings

image
image
image

### CODE REVIEW

File: components/MechPage/MechAgentMetrics.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <Card className={`${CARD_BG} max-h-[250px] p-16`}>
+        <Image
+          alt="Mech metrics"
+          src="/images/optimus-page/optimus-metrics.svg"
+          height={48}
+          width={48}
+          className="mb-6 mx-auto"
+        />
+        <p className="text-slate-500">
+          Olas Mech agent economy metrics are coming soon.
```
Comment (truemiller) [inline components/MechPage/MechAgentMetrics.jsx:21]:
```suggestion
          Olas Mech agent economy metrics coming soon.
```

File: components/MechPage/WhatIsOlasMech.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        src="/images/mech-page/mech-diagram.png"
+        alt="Mech diagram"
+        width={920}
+        height={595}
+        className="mx-auto"
+      />
+    </Card>
+    <p className="mb-3">
+      The Mech economy sells AI services to other agents via the blockchain.
+      Building on this concept, Olas Mech agent economy provides AI intelligence
```
Comment (truemiller) [inline components/MechPage/WhatIsOlasMech.jsx:25]:
```suggestion
      Building on this concept, the Olas Mech agent economy provides AI intelligence
```

File: components/MechPage/WhatIsOlasMech.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      for other autonomous AI agents or agent economies by listening for
+      on-chain requests and performing the needed actions off-chain in exchange.{' '}
+    </p>
+    <p className="mb-20">
+      This system creates a seamless marketplace for AI capabilities, where
+      agents can request and receive specialized services through secure,
+      decentralized transactions.
+    </p>
```
Comment (truemiller) [inline components/MechPage/WhatIsOlasMech.jsx:33]:
```suggestion
    <p className="mb-3">
      The Olas Mech economy sells AI services to other agents via the blockchain.
      Building on this concept, it provides intelligence
      to other autonomous AI agents and agent economies by listening for
      on-chain requests and performing the needed actions off-chain.{' '}
    </p>
    <p className="mb-20">
      This system creates a seamless marketplace for AI capabilities, where
      agents can request and receive specialized services through secure,
      decentralized transactions.
    </p>
```

File: components/MechPage/WhyOlasMech.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { SECTION_BOX_CLASS, SUB_HEADER_LG_CLASS } from 'common-util/classes';
+import { InfoCardList } from 'components/InfoCardList';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { BicepsFlexed, HandCoins, Sparkles } from 'lucide-react';
+
+const list = [
+  {
+    title: 'Specialisation',
```
Comment (truemiller) [inline components/MechPage/WhyOlasMech.jsx:8]: currently correct from UK standpoint 😄 though Americans will 🤮 
```suggestion
    title: 'Specialization',
```

File: components/MechPage/MechAgentMetrics.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+const CARD_BG =
+  'border-1.5 border-gray-200 rounded-2xl p-6 bg-gradient-to-t from-[#F2F4F7] to-white';
+
+export const MechAgentMetrics = () => (
+  <SectionWrapper customClasses={`${SECTION_BOX_CLASS} lg:py-14`}>
+    <div className="max-w-[750px] mx-auto text-center">
+      <Card className={`${CARD_BG} max-h-[250px] p-16`}>
+        <Image
+          alt="Mech metrics"
+          src="/images/optimus-page/optimus-metrics.svg"
```
Comment (truemiller) [inline components/MechPage/MechAgentMetrics.jsx:15]: this is broken on preview
image

File: pages/agent-economies/mech.jsx [hunk 6]
```diff
@@ -1,12 +1,18 @@
 import PageWrapper from 'components/Layout/PageWrapper';
+import { MechAgentMetrics } from 'components/MechPage/MechAgentMetrics';
 import { MechHero } from 'components/MechPage/MechHero';
+import { WhatIsOlasMech } from 'components/MechPage/WhatIsOlasMech';
+import { WhyOlasMech } from 'components/MechPage/WhyOlasMech';
 import Meta from 'components/Meta';
 
 const Mech = () => (
   <PageWrapper>
     <Meta pageTitle="Mech" description="AI intelligence for agent economies" />
```
Comment (truemiller) [inline pages/agent-economies/mech.jsx:10]: `intelligence` is redundant as mentioned above..
would go for: 
- services
- marketplace
etc ...
```suggestion
    <Meta pageTitle="Mech" description="AI services for agent economies" />
```

File: components/MechPage/WhyOlasMech.jsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  {
+    title: 'Scale',
+    icon: <BicepsFlexed color="#606F85" />,
+    desc: 'Thanks to Mech marketplace, Mechs can scale horizontally, with new Mechs being added by the Olas community to serve a diverse set of use cases',
+  },
+  {
+    title: 'Fee capture',
+    icon: <HandCoins color="#606F85" />,
+    desc: 'Mech marketplace offers a fee capture mechanism for the Olas protocol.',
```
Comment (truemiller) [inline components/MechPage/WhyOlasMech.jsx:20]:
```suggestion
    desc: 'The Mech marketplace offers a fee capture mechanism for the Olas protocol.',
```

File: components/MechPage/MechAgentMetrics.jsx (hunk 5 above)
Comment (Tanya-atatakai) [reply components/MechPage/MechAgentMetrics.jsx:15]: @atepem maybe if it's used on both pages, can name the icon just "metrics" and place in `/images`

**Review by truemiller — APPROVED** [review]

**Review by Tanya-atatakai — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**